core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/screen/ScreenCollector.java

```diff
@@ -56,12 +62,27 @@
 
   private final CollectorProperties properties;
 
+  private String excludeCssSelector;
+
   ScreenCollector(CollectorProperties properties, WebDriver webDriver, ArtifactsDAO artifactsDAO) {
     this.properties = properties;
     this.webDriver = webDriver;
     this.artifactsDAO = artifactsDAO;
   }
 
+  private List<ExcludedElement> getExcludeElementsFromWebElements(
+      List<WebElement> webElements) {
+    List<ExcludedElement> excludeExcludedElements = new LinkedList<>();
```

Comment:
Please use `ArrayList` here to have better performance. You may estimate it's size with `webElements.size()`.